website/pages/blog/open-source-graphql-security.mdx

```diff
@@ -0,0 +1,242 @@
+years.
+
+### Shared Safety with Public Security Resources
+
+Let’s talk about public security resources, like the Common Vulnerabilities and Exposures
+([CVE](https://cve.mitre.org/)) system. It's like a public library of cybersecurity issues,
+describing and referencing known vulnerabilities for everyone to see and learn from. This database
+gives developers and cybersecurity pros a common ground to talk about cybersecurity threats,
+spreading the word about new dangers fast and wide. It’s our shared language and memory for
+cybersecurity issues, helping us all to better safeguard our digital spaces from known threats.
+Making sure that vulnerabilites are made public can seem counter-intuitive at first glance, but is a
+massive need to ensure that fixes or other security measures are deployed rapidely and widely.
+
+### Open Source: Our Collective Eye in Cybersecurity
+
+Open source in cybersecurity shines a light on possible threats and also hands us the tools and
+methods to tackle them. It’s a world where problems are solved together, providing a stage where
+vulnerabilities are spotted, discussed, and fixed openly and transparently. This cooperative space
+crafts tools and strategies available to everyone, ensuring the fruits of new discoveries and
+developments are shared with the whole digital world.
+
+Moreover, open-source security tools allow organizations, big or small, to protect their operations
+without starting from scratch. These tools lay down a solid base upon which any company can build on
+and mold their cybersecurity strategies as per their needs.
+
+Open offensive security tools, for instance the ones packed into the popular Kali linux distribution
+or Parrot OS, are also benefic to the cyber world, as making the attack strategies public helps to
+build more secure applications and infrastructures.
+
+### Open Source and Cybersecurity: A Perfect Pair
+
+In a field like cybersecurity, where dangers morph quickly, having shared knowledge, united
+alertness, and available-to-all tools become an essential shield against potential attacks. The
+marriage of open source and cybersecurity offers a lively platform for ongoing learning, changing,
+and sharing attack and defense mechanisms, crafting a global, united front against cyber threats.
+
+Moving forward, we'll dive into how this perfect pairing of open source and cybersecurity is
+crucial, not just relevant, for GraphQL and API security. We'll highlight practical tools and
+strategies you can use today to protect your applications.
+
+## Open Source GraphQL Security Tools
+
+There are many GraphQL open-source tools available to help developers and businesses defend against
+possible cybersecurity threats. From defensive measures that shield sensitive data to offensive
+tools aimed at identifying vulnerabilities, the open-source community has build invaluable resources
+to cover a wide variedy of cyber-security needs.
```

Comment:
Something very big that's missing for me here are best practices.
For example, first thing we always say to any GraphQL developer, is not to expose an open GraphQL API but to always use persisted queries, which eliminate most threads from the get go!

I would put that at the top of any list

Reply:
Best practices are mentionned via an article later on but I can put them higher here. Will reorganize